framer/scripts/framerConfig.js

```diff
@@ -130,6 +130,8 @@ export const componentSettings = {
       // FIXME: `Union`
```

Comment:
This should be corrected in `docs/pages/api-docs/[component].json`? If so, they are all as `union` (`variant`, `color`, `size`). If so, should we delete this comment?

Reply:
It's for framer, it doesn't matter